### PR TITLE
dlb: update DLB to v7.7.0

### PR DESCRIPTION
**Pull request by bart0sh**

### CODE REVIEW

**Review by hj-johannes-lee — APPROVED** [review]
LGTM :)

File: cmd/dlb_plugin/README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ If you configure SR-IOV/VF (virtual functions), continue the following configura
 Patch dpdk sources to work with DLB:
 ```bash
 $ wget -q https://fast.dpdk.org/rel/dpdk-20.11.4.tar.xz -O- | tar -Jx
```
Comment (hj-johannes-lee) [inline cmd/dlb_plugin/README.md:36]: Maybe not your mistake, but I guess this should be also changed?

**Review by hj-johannes-lee — CHANGES_REQUESTED** [review]

File: cmd/dlb_plugin/README.md (hunk 1 above)
Comment (bart0sh) [reply cmd/dlb_plugin/README.md]: Thank you for pointing out to this. Should be fixed now.

**Comment by bart0sh** [comment]
@hj-johannes-lee Can you merge this PR please?

**Review by hj-johannes-lee — APPROVED** [review]

**Review by hj-johannes-lee — APPROVED** [review]

**Comment by hj-johannes-lee** [comment]
@bart0sh Hi, Idk what approval it is asking. Merging is blocked though I approved.

**Review by mythi — APPROVED** [review]

**Pull request merged**